src/Schema/AST/DocumentAST.php

```diff
@@ -134,6 +134,16 @@ public function objectTypes()
         return $this->definitionsByType(ObjectTypeDefinitionNode::class);
     }
 
+    /**
+     * Get all definitions for input types.
+     *
+     * @return Collection
+     */
+    public function inputTypes()
```

Comment:
Maybe we should clean up the naming with these functions. We always have `FooBarDefinitionNode` as the underlying type.